src/ontology/demo-edit.owl

```diff
@@ -1,43 +1,46 @@
-
 Prefix(:=<http://purl.obolibrary.org/obo/demo.owl#>)
+Prefix(dce:=<http://purl.org/dc/elements/1.1/>)
 Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
 Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
 Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
 Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
 Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
-Prefix(dce:=<http://purl.org/dc/elements/1.1/>)
 Prefix(dcterms:=<http://purl.org/dc/terms/>)
 
 
 Ontology(<http://purl.obolibrary.org/obo/demo.owl>
-
-Import(<http://purl.obolibrary.org/obo/demo/imports/ro_import.owl>)
-
+Import(<http://purl.obolibrary.org/demo/modules/test.owl>)
+Import(<http://purl.obolibrary.org/obo/demo/imports/bco_import.owl>)
 Import(<http://purl.obolibrary.org/obo/demo/imports/bfo_import.owl>)
-
-Import(<http://purl.obolibrary.org/obo/demo/imports/pato_import.owl>)
-
 Import(<http://purl.obolibrary.org/obo/demo/imports/envo_import.owl>)
-
-Import(<http://purl.obolibrary.org/obo/demo/imports/bco_import.owl>)
-
-
-
+Import(<http://purl.obolibrary.org/obo/demo/imports/pato_import.owl>)
+Import(<http://purl.obolibrary.org/obo/demo/imports/ro_import.owl>)
+Annotation(dce:description "None")
 Annotation(dce:title "Demonstration Ontology")
 Annotation(dcterms:license <https://creativecommons.org/licenses/unspecified>)
-Annotation(dce:description "None")
 
 Declaration(Class(<http://purl.obolibrary.org/obo/DEMO_00000000>))
+Declaration(AnnotationProperty(<http://purl.obolibrary.org/obo/demo#Physiology>))
+Declaration(AnnotationProperty(dce:description))
+Declaration(AnnotationProperty(dce:title))
+Declaration(AnnotationProperty(dcterms:license))
+############################
+#   Annotation Properties
+############################
+
+# Annotation Property: <http://purl.obolibrary.org/obo/demo#Physiology> (<http://purl.obolibrary.org/obo/demo#Physiology>)
+
+SubAnnotationPropertyOf(<http://purl.obolibrary.org/obo/demo#Physiology> <http://www.geneontology.org/formats/oboInOwl#SubsetProperty>)
```

Comment:
You don't want an annotation property for `DEM:Physiology`. You want to use the `oboInOwl:inSubset` annotation property with a value of `DEM:Physiology`. See this line of the Uberon example from ROBOT:

https://github.com/ontodev/robot/blob/master/docs/examples/uberon_module.owl#L281

Reply:
Thank you @jamesaoverton I think I have it figured out now. I had previously tried it as you had suggested. The the issue was that I was trying to filter from the `edit.owl` file, however all the classes I was trying to filter aren't actually in their as they are direct imports from the robot created module only actually exist there. Protege direct imports can be a little misleading.

Thanks again!